src/utils/colorManipulation.ts

```diff
@@ -104,103 +109,108 @@ export function getAverageColor(
-    );
-    return EMPTY_PIXEL;
-  }
-
-  const truePixelIdx = (row * frame.width + col) * 4;
-  return frame.data.slice(truePixelIdx, truePixelIdx + 4);
-}
-
-const EMPTY_PIXEL = [0, 0, 0, 255];
-
 /**
  * - A function that is used to smooth the color transition between divisions
  * @param frame is the default canvas that holds the divison width/length colors]
  * - Defines the boundaries for region convolution
- * @param startRow 
- * @param startCol 
- * @param endRow 
- * @param endCol 
+ * @param startRow
+ * @param startCol
+ * @param endRow
+ * @param endCol
  * @param kernel_size is how large your kernel convolution size is (higher = laggier but smoother transition)
- * @returns an ImageData 
+ * @returns an ImageData
  */
 
 export function regionConvolution(
-    frame: ImageData,
-    startRow: number,
-    startCol: number,
-    endRow: number,
-    endCol: number,
-    kernel_size: number
+  frame: ImageData,
+  startRow: number,
+  startCol: number,
+  endRow: number,
+  endCol: number,
+  kernel_size: number
 ): ImageData {
-    const frameCopy = new Uint8ClampedArray(frame.data.length);
-    let layers = Math.floor(kernel_size / 2);
-    for(let row = startRow; row < endRow; row++) { 
-        for(let col = startCol; col < endCol; col++) {
-            let currentPixel = frame.data[(row * frame.width + col) * 4];
-
-            const result = convolveRegion(row, col, layers, frame, startRow, startCol, endRow, endCol);
-
-            frameCopy[(currentPixel++)] = result[0] / kernel_size * kernel_size;
-            frameCopy[(currentPixel++)] = result[1] / kernel_size * kernel_size;
-            frameCopy[(currentPixel++)] = result[2] / kernel_size * kernel_size;
-            frameCopy[(currentPixel++)] = result[3] / kernel_size * kernel_size;
-        }
+  const frameCopy = new Uint8ClampedArray(frame.data.length);
+  let layers = Math.floor(kernel_size / 2);
+  for (let row = startRow; row < endRow; row++) {
+    for (let col = startCol; col < endCol; col++) {
+      let currentPixel = frame.data[(row * frame.width + col) * 4];
+
+      const result = convolveRegion(
+        row,
+        col,
+        layers,
+        frame,
+        startRow,
+        startCol,
+        endRow,
+        endCol
+      );
+
+      frameCopy[currentPixel++] = (result[0] / kernel_size) * kernel_size;
+      frameCopy[currentPixel++] = (result[1] / kernel_size) * kernel_size;
+      frameCopy[currentPixel++] = (result[2] / kernel_size) * kernel_size;
+      frameCopy[currentPixel++] = (result[3] / kernel_size) * kernel_size;
     }
+  }
 
-    return new ImageData(frameCopy, (endRow - startRow), (endCol - startCol))
+  return new ImageData(frameCopy, endRow - startRow, endCol - startCol);
 }
 
 export function convolveRegion(
-    row: number,
-    col: number, 
-    layers: number, 
-    frame: ImageData,
-    startRow: number,
-    startCol: number,
-    endRow: number,
-    endCol: number
+  row: number,
```

Comment:
sorry, just realizing that my prettier is sent to 2 space default bc of work :(

Reply:
i can change if you feel strongly, but i personally prefer the 2 space tab over 4 space